packet/templates/packet.html

```diff
@@ -26,6 +26,9 @@ <h3>{{ freshman.name }}</h3>
                 <div class="col">
                     <h5>Signatures: {{ signed }}/{{ required }}</h5>
                 </div>
+                <div class="col">
```

Comment:
Can you please add a screenshot showing how this is displayed?

Reply:
Can't do that.